moveit_core/collision_detection_bullet/README.md

```diff
@@ -3,6 +3,3 @@ To use Bullet as a collision checker you can manually switch to Bullet using the
 
 ### Speed Benchmarks
 For speed comparisons to FCL please see (relative link to README of benchmark script will be added).
-
-### Current Limitations
-The collision detection using Bullet is not thread safe as the internal collision managers are `mutable` members of the collision environment.
```

Comment:
Instead of removing this section, could you please change the sentence to say that the bullet collision environment does not profit from parallel collision checking. The mutex gets rid of undefined behavior in parallel use, but it does not magically give you a speedup if you *do* parallelize.

Reply:
done